deploy-nagios/tasks/main.yml

```diff
@@ -32,6 +32,21 @@
     nagios_image_version: "{{ nagios_image_def[0][0].image }}"
 
 -
+  name: "Set value for platforms"
+  set_fact:
+    digger_platforms: "{{ digger_platforms if digger_platforms is defined and digger_platforms != '' else 'android,ios' if groups['macos'] is defined and groups['macos'] | length > 0 else 'android' }}"
+
+- name: "oc get dc nagios"
+  command: oc get dc nagios
+  register: nagios_dc
+  failed_when: "'NotFound' not in nagios_dc.stderr"
```

Comment:
This just fails task if nagios is there, I don't think we want that.

Reply:
@matskiv yes I had the change locally for this. `and node_count.stdout == ''" ` should be added to the failed_when